### DISCUSSION TITLE
Is there a way to add default locale to the route when no locale is specified?

### DISCUSSION THREAD

**Discussion by binums** [vercel/next.js#19015] [Help] [answered]
I've upgraded to next@10.0.0 and am using internationalized routes with sub-path routing.

next.config.js
```module.exports = {
 i18n: {
      locales: ['en-US', 'es-US'],
      defaultLocale: 'en-US',
    },
}
```

I have 2 locales and `/en-US` is my default locale. Now, is there a way for me automatically redirect to `/en-US` when no locale is provided. Eg, `<domain>/something` will have to be redirected to `<domain>/en-US/something`

**Comment by AssisrMatheus** [comment, marked as answer]
I think this is related to https://github.com/vercel/next.js/discussions/18419
